app/views/unattended/provisioning_templates/snippet/ca_registration.erb

```diff
@@ -0,0 +1,39 @@
+<%#
+kind: snippet
+name: ca_registration
+model: ProvisioningTemplate
+snippet: true
+description: |
+  This template is used for updating Foreman's CA on hosts that are registered by Katello.
+  It replaces the CA used by subscription-manager and adds the CA to trusted anchors.
+-%>
+
+<% if plugin_present?('katello') -%>
+  # Define the path to the Katello server CA certificate
+  KATELLO_SERVER_CA_CERT=/etc/rhsm/ca/katello-server-ca.pem
+
+  # If katello ca cert file exists on host, update it and make sure it's in trust anchors
+  if [ -f "$KATELLO_SERVER_CA_CERT" ]; then
+    cat << EOF > "$KATELLO_SERVER_CA_CERT"
+<%= foreman_server_ca_cert %>
+EOF
+
+    if [ -f /etc/debian_version ]; then
+      CA_TRUST_ANCHORS=/usr/local/share/ca-certificates/
+    else
+      CA_TRUST_ANCHORS=/etc/pki/ca-trust/source/anchors
+    fi
+
+    # Add the Katello CA certificate to the system-wide CA certificate store
+    if [ -d $CA_TRUST_ANCHORS ]; then
+      if [ -f /etc/debian_version ]; then
+        cp $KATELLO_SERVER_CA_CERT $CA_TRUST_ANCHORS
+        update-ca-certificates
+      else
+        update-ca-trust enable
```

Comment:
What does `enable` do? I don't see that in `man update-ca-trust`. In fact, the whole script just disregards the parameter.
```suggestion
```

Reply:
I have copied an existing template, but removed it now, since it doesn't make sense anymore.